pkgs/tools/typesetting/tex/mftrace/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, lib, makeWrapper, autoconf, python3, fontforge, potrace }:
+stdenv.mkDerivation rec {
+  pname = "mftrace";
+  version = "1.2.20";
+
+  # https://lilypond.org/download/sources/mftrace/mftrace-1.2.20.tar.gz
+  # is incomplete, fetch repo and use autoconf instead
+  # see https://github.com/hanwen/mftrace/issues/13
+  src = fetchFromGitHub {
+    owner = "hanwen";
+    repo = "mftrace";
+    rev = "release/${version}";
+    sha256 = "02ik25aczkbi10jrjlnxby3fmixxrwm2k5r4fkfif3bjfym7nqbc";
+  };
+
+  # the potrace/autotrace detection is done at runtime
+  # and can be omitted at build time
+  patches = [ ./disable-build-time-trace-detection.patch ];
+
+  nativeBuildInputs = [ makeWrapper autoconf python3 ];
+
+  buildInputs = extraBuildInputs;
+
+  # at runtime, mftrace needs:
+  # - a tex distribution containing the fonts and the binaries
+  #   kpsewich, mf (metafont), t1asm, mktextfm
+  # - one of potrace (preferred), autotrace
+  # - one of fontforge (preferred), ghostscript
+  extraBuildInputs = [ fontforge potrace ];
+
+  preConfigure = ''
+    autoconf
+  '';
```

Comment:
We should probably use autoreconfHook here to prevent issues when updating autoconf etc.